### DISCUSSION TITLE
How to access theia from browser use network ip address

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-theia/theia#13526] [Q&A] [answered]
hi,
   1. I use yarn browser start to start my theia blueprint, and access it in the browser using 127.0.0.1:3000, it is ok
   2. But when I use my local computer's actual network IP address, for example, 192.168.1.100:3000, I can not open the Theia browser. 
   3. does theia bind 127.0.0.1 default? how could I do to make it access theia backend using the actual IP address

**Comment by bodnarlajos** [comment, marked as answer]
The docker image start with this "node /home/theia/src-gen/backend/main.js /home/project --hostname=0.0.0.0"
I think you should add or configure somewhere the --hostname parameter.

**Reply by wss29** [reply]
where could I set --hostname? in package.json?

**Reply by msujew** [reply]
It's an argument of the Theia server script. As mentioned in the previous command:

```
node /home/theia/src-gen/backend/main.js /home/project --hostname=0.0.0.0
```

**Reply by wss29** [reply]
hi
  I add it in package.json
"start": "theia start --plugins=local-dir:../../plugins --hostname=0.0.0.0", it works now thanks
